fetch.bs

```diff
@@ -1021,14 +1044,27 @@ one or more event listeners are registered on an {{XMLHttpRequestUpload}} object
 which is "<code>omit</code>", "<code>same-origin</code>", or
 "<code>include</code>". Unless stated otherwise, it is "<code>omit</code>".
 
-<p class="note no-backref"><a for=/>Request</a>'s
-<a for=request>credentials mode</a> controls the flow of
-<a for=/>credentials</a> during a <a for=/>fetch</a>. When
-<a for=/>request</a>'s <a for=request>mode</a> is
-"<code>navigate</code>", its
-<a for=request>credentials mode</a> is assumed to be
-"<code>include</code>" and <a for=/>fetch</a> does not currently account
-for other values. If <cite>HTML</cite> changes here, this standard will need corresponding changes.
+<div class="note no-backref">
+ <dl>
+  <dt>"<code>omit</code>"
+  <dd>Used to ensure credentials are not included with this request.
```

Comment:
"Excludes credentials from the request". In the sense that it doesn't really ensure it, that's just what happens.